components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/resources/web/call-mediator/update-mediator.jsp

```diff
@@ -79,6 +80,9 @@
         callMediator.setEndpoint(ep);
     }
 
+    blocking=Boolean.parseBoolean(request.getParameter("mediator.call.blocking"));
```

Comment:
We can declare the 'blocking' variable here.
boolean blocking = Boolean.parseBoolean(request.getParameter("mediator.call.blocking"));